doc/source/metadata/Probabilities.rst

```diff
@@ -1,42 +1,38 @@
 .. _prob-section:
 
-Probabilistic Information
-=========================
+Probability Distribution
+========================
 
 .. contents:: Contents
     :depth: 3
 
 Context
 -------
 
-As IMPROVER is inherently probabalistic,
-it seems appropriate to have a section specifically focussed
-on the represntation of probabalistic information in the metadata.
-This section also looks at the use of the `CF Metadata Conventions`_ 
-cell methods which are also heavily used within IMPROVER.
+As IMPROVER is inherently probabilistic,
+it seems appropriate to have a section specifically focused
+on the representation of probabality distributions in the metadata.
```

Comment:
```suggestion
on the representation of probability distributions in the metadata.
```
Sorry, I must have missed this one on the first pass!

Reply:
corrected.